.concourse/pipeline.yaml.gomplate

```diff
@@ -808,6 +809,108 @@ jobs:
 {{- end }}
 {{ $previousTest = (printf "cf-acceptance-tests-%s-%s" $cfScheduler $branch) }}
 
+- name: cats-internetless-{{ $cfScheduler }}-{{ $branch }}
+  public: true
+  plan:
+  - get: kubecf-{{ $branch }}
+    passed:
+    - {{ $previousTest | quote }}
+    trigger: true
+    version: "every"
```

Comment:
did we decide to remove all `version: every`, of was that for a different resource?

Reply:
it was a different one but makes sense to remove it here too